packages/matchbox/src/components/Toggle/styles.js

```diff
@@ -0,0 +1,68 @@
+import { tokens } from '@sparkpost/design-tokens';
+import styled from 'styled-components';
+
+export const toggle = props => `
+  display: inline-block;
+  position: relative;
+  height: 1.25rem;
+  width: 2.25rem;
+
+  opacity: ${props.disabled ? '0.6' : '1'}
+
+  &:hover {
+    cursor: ${props.disabled ? 'not-allowed' : 'pointer'};
```

Comment:
If you wanted, you could just move this outside of the `:hover`? I think it would still work and decrease specificity of the rule slightly.